public/timeline-configuration/state-helpers.test.js

```diff
@@ -79,7 +79,7 @@ test("rawIssuesRequestData", function(assert){
 test("derivedIssuesRequestData", async function(assert){
     const rawIssuesRequestData = value.with({
         issuesPromise: Promise.resolve([{key: "TEST-123", fields: {
-            
+            "Issue Type": {hierarchyLevel: 7},
```

Comment:
Needed to add this to get the test passing... Is that something to be concerned with @justinbmeyer ?